### PR TITLE
added notification off docs

### PR DESCRIPTION
**Pull request by captain-Akshay**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@nwanduka., how did @captain-Akshay do?

File: content/en/meshmap/designer/comments/comment-notificationBell.png
Comment (leecalcote) [inline content/en/meshmap/designer/comments/comment-notificationBell.png:1]: Thanks for this. Two issues:

1) poor quality screenshot.
2) let's have pleasant, positive sample interactions.

File: content/en/meshmap/designer/comments/_index.md [hunk 1]
```diff
@@ -68,6 +68,12 @@ After a comment has been resolved, there might be situations where you need to r
 
 <img src="./comments-unresolved.gif" alt="Unresolving comments in designer" width="600">
 
+### Mute Comment Notifications
+
+Customize your notification preferences to mute emails for comments on your design. When this feature is enabled, you won't receive email notifications for new comments on your design. This can be useful if you want to temporarily pause notifications or reduce email clutter. Follow the steps below to mute comment notifications
```
Comment (leecalcote) [inline content/en/meshmap/designer/comments/_index.md:73]: You don't "mute emails", you "mute notifications" or "mute email notifications".

Comment (leecalcote) [inline content/en/meshmap/designer/comments/_index.md:73]: "with this feature enabled"... no. Please change to "With notifications for new comments silenced, ..."

Comment (leecalcote) [reply content/en/meshmap/designer/comments/_index.md:73]: "you won't receive email notifications for new comments on your design." - what about existing comments and new threaded added?

@captain-Akshay, buckle down a description of the logic here - describe the behavior in various scenarios / under various conditions.

**Review by leecalcote — CHANGES_REQUESTED** [review]
@Yashsharma1911 , it's unclear according to this documentation whether this silence notifications button is a per comment silencer for a silencer for all comments... in a design or for my entire user account? Does this setting only affect me or does it affect other users of this design? Do I need to own the design in order to use this? 

What happens if I muted, people make comments, I unmute will I receive historical notifications for comments that were placed while I had silence on?

**Comment by leecalcote** [comment]
@captain-Akshay you can do better. And I expect you to do better.

**Comment by leecalcote** [comment]
@nwanduka, it seems that you might have missed this review. The comments that I've made could / should be made by you as either the first reviewer or the writer of this PR.

**Comment by nwanduka** [comment]
> @nwanduka, it seems that you might have missed this review. The comments that I've made could / should be made by you as either the first reviewer or the writer of this PR.

I started a review and didn't realize I hadn't submitted it. I was honestly confused when I kept getting the request to review the PR. All the while I was thinking to myself, "is no one else seeing the comment I left?" Now I know why 🤦🏽‍♀️
image

File: content/en/meshmap/designer/comments/comment-notificationBell.png
Comment (Yashsharma1911) [reply content/en/meshmap/designer/comments/comment-notificationBell.png:1]: +1

**Comment by Yashsharma1911** [comment]
> @Yashsharma1911 , it's unclear according to this documentation whether this silence notifications button is a per comment silencer for a silencer for all comments... in a design or for my entire user account? Does this setting only affect me or does it affect other users of this design? Do I need to own the design in order to use this?
> 
> What happens if I muted, people make comments, I unmute will I receive historical notifications for comments that were placed while I had silence on?

These are good questions and yes, this doc doesn't answer all of these questions, @captain-Akshay I'll help here with updating this doc

**Comment by iArchitSharma** [comment]
@Yashsharma1911 @captain-Akshay @nwanduka added draft label please remove this when your finished with your changes

**Comment by captain-Akshay** [comment]
image

**Review by nwanduka — APPROVED** [review]
Quite an improvement from the first draft. The content looks good to me 👍🏽.

How about the image? Did you change the comment in the screenshot to a more positive one? If that's done, then I think we're set to get this merged.

File: content/en/meshmap/designer/comments/_index.md [hunk 2]
```diff
@@ -68,6 +68,28 @@ After a comment has been resolved, there might be situations where you need to r
...
+  -  *Mentions in Other Designs:* You'll still be notified if someone mentions you in new or existing comment threads on other designs in your portfolio.
+  -  *New Comment Threads:* Muting only applies to the specific comment thread it's activated on. New threads on the same design will trigger notifications as usual.
+
+{{< alert title="Remember">}}
+- While email alerts are muted, you can still access and view all comments on the design at any time.
+- You'll never miss an important mention, as notifications for mentions outside the muted thread remain active.
+{{< /alert >}}
+
+
+<img src="./comment-notificationBell.png" alt="Turn Off notification from comments in designer" style="width:auto">
```
Comment (iArchitSharma) [inline content/en/meshmap/designer/comments/_index.md:91]: can you please place the image above **What Muting Affects:**

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
It's good for merging once the image is repositioned.

**Review by iArchitSharma — APPROVED** [review]
did it myself, good to merge!

**Pull request merged**